modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-email-otp-web/src/main/java/com/liferay/multi/factor/authentication/email/otp/web/internal/portlet/action/SendMFAEmailOTPMVCResourceCommand.java

```diff
@@ -95,7 +95,7 @@ public boolean serveResource(
 
 			if (userId == null) {
 				if (_log.isWarnEnabled()) {
-					_log.warn("User id is not in the session");
```

Comment:
hey @brianchandotcom, I copied this snippet from the portal: 
https://github.com/liferay/liferay-portal/blob/b3173da81b62430f7150b695d5e944b224279580/portal-impl/src/com/liferay/portal/servlet/PortalSessionDestroyer.java#L67